doc_admin/AdminManual.adoc

```diff
@@ -132,12 +132,13 @@ NOTE: The information needed to use the OH software is not included in this manu
 
 == 2 Installation
 
-Open Hospital is a client application that can be used in two different ways: 
+Open Hospital is a client application that can be used in three different ways: 
 
 * <<client,2.1 OH - CLIENT mode>>: multi computers setup, with a central database (requires a little IT knowledge)
 * <<portable,2.2 OH - PORTABLE mode>>: single computer setup, with inbuilt databases (click and run)
+* <<server,2.3 OH - SERVER mode (portable)>>: multi computer setup, with the central database created by Open Hospital itself (click and run)
```

Comment:
Line 137: `multi computers setup` -> `multi computer setup`
Line 138: `with inbuilt databases` -> `with the database on the same computer`
Line 139: `created by Open Hospital itself` -> `created by the Open Hospital script`

Reply:
Done, thanks!